### PR TITLE
sstable tool support for the modules

### PR DESCRIPTION
**Pull request by sylvestor88**

### CODE REVIEW

File: test/validation/org/apache/cassandra/bridges/ArchiveClusterLogs.java [hunk 1]
```diff
@@ -147,4 +147,11 @@ public static boolean countErrors(String errorLogs)
         else
             return false;
     }
+
+    public static String[] metaDataFilesFormatter(String files)
+    {
+        files = files.substring(1, files.length()-1);
```
Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/bridges/ArchiveClusterLogs.java:153]: There's no need for the second argument here.

Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/bridges/ArchiveClusterLogs.java:153]: Why are we removing the first character?

**Comment by ptnapoleon** [comment]
The SSTableMetadata class shouldn't be in bridges. Put it in its own tools package. What is even referencing that class?

File: test/validation/org/apache/cassandra/bridges/ctoolbridge/CToolBridge.java [hunk 2]
```diff
@@ -239,4 +239,23 @@ public void nodeTool(int node, String command, String arguments)
...
     }
+
+    public void ssTableSplit(int node, String options, String keyspace_path)
+    {
+        String fullCommand;
+        if (options == "")
+        {
+            fullCommand = "ctool run " + DEFAULT_CLUSTER_NAME + " " + node + "\'/home/automaton/cassandra*/tools/bin/sstablesplit /home/automaton/cassandra*/data/data/" + keyspace_path + "\'";
+        }
+        else{
```
Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/bridges/ctoolbridge/CToolBridge.java:251]: Put your braces on new lines.

File: test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java [hunk 3]
```diff
@@ -279,4 +297,44 @@ public void removeLiveCluster(String temp_dir)
...
+        String fullCommand;
+        if(options == "")
+        {
+            fullCommand = "ccm node" + node + " sstablesplit" + " --config-dir=" + ccmDir;
+        }
+        else
+        {
+            fullCommand = "ccm node" + node + " sstablesplit " + options + " --config-dir=" + ccmDir;
+        }
+        executeAndPrint(fullCommand);
```
Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java]: You're going to be duplicating --config-dir here, no? Same as in ssTableMetaData() at line 327

File: test/validation/org/apache/cassandra/bridges/ArchiveClusterLogs.java (hunk 1 above)
Comment (sylvestor88) [inline test/validation/org/apache/cassandra/bridges/ArchiveClusterLogs.java:153]: Removing first '[' and last ']' characters from the list getting from ccm getsstables

Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/bridges/ArchiveClusterLogs.java:153]: Okay. Disregard my comments here then.

File: test/validation/org/apache/cassandra/bridges/ctoolbridge/CToolBridge.java (hunk 2 above)
Comment (sylvestor88) [inline test/validation/org/apache/cassandra/bridges/ctoolbridge/CToolBridge.java:251]: Done

File: test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java (hunk 3 above)
Comment (sylvestor88) [inline test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java]: I am not really clear about this question, duplicating?

Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java]: Explained offline, but you're passing in a command with `--config-dir` to a method that appends `--config-dir`.

**Comment by ptnapoleon** [comment]
Once you've removed SSTableMetadata.java, and made the execute changes, i'll merge

**Comment by sylvestor88** [comment]
Made execute changes and removed SSTableMetaData.java

**Pull request merged**